Public/Sdk/Public/Transformers/Transformer.Write.dsc

```diff
@@ -43,4 +44,35 @@ namespace Transformer {
     /** The content of a file that can be written using writeFile. */
     @@public
     export type FileContent = PathFragment | Path | (PathFragment | Path)[];
+
+    @@public
+    export interface WriteDataArguments extends CommonWriteArguments {
+        /** The contents to write */
+        contents: Data;
+    }
+
+    @@public
+    export interface WriteAllLinesArguments extends CommonWriteArguments {
+        /** The lines to write */
+        lines: Data[];
+    }
+
+    @@public
+    export interface WriteAllTextArguments extends CommonWriteArguments {
+        /** The text to write */
+        text: string;
+    }
+

```

Comment:
```suggestion
```

Reply:
If you use the GitHub web UI, you can commit this suggestion right from the UI ;)

Reply:
yeah, but I am still struging to be productive in the UI, in my view the github sorting is wrong and it doesn't provide me any details on what changed since I last viewed. I'm still longing for Azure Repo's web interface so I'll probably stay in codeflow for a while.

---
In reply to: [272251496](https://github.com/Microsoft/BuildXL/pull/82#discussion_r272251496) [](ancestors = 272251496)